tests/test_eload_backlog.py

```diff
@@ -47,7 +48,7 @@ def test_fill_in_config(self):
             'brokering': {
                 'vcf_files': {expected_vcf: {'index': expected_index}},
                 'ena': {
-                    'hold_date':  '2021-01-01+01:00',
+                    'hold_date':  datetime(2021, 1, 1, 0, 0, tzinfo=timezone(timedelta(seconds=3600))),
```

Comment:
I'm expecting the hold date to be written as `2021-01-01 0:0:0` if it is a datetime object. I think it is down to the use of `str()` or `repr()` 
When the hold date is provided by the metadata [here](https://github.com/EBIvariation/eva-submission/blob/e0d0e5e080b81e8a2ff70ed677113c59b60794c5/eva_submission/ENA_submission/xlsx_to_ENA_xml.py#L291) we get it from the XLS reader which [should return a python datetime](https://openpyxl.readthedocs.io/en/latest/datetime.html) we also create the hold date if it does not exist [here](https://github.com/EBIvariation/eva-submission/blob/e0d0e5e080b81e8a2ff70ed677113c59b60794c5/eva_submission/ENA_submission/xlsx_to_ENA_xml.py#L293) that also create a datetime.
All of those are set [here](https://github.com/EBIvariation/eva-submission/blob/e0d0e5e080b81e8a2ff70ed677113c59b60794c5/eva_submission/eload_brokering.py#L67) and results in the string representation using`str()`
Not sure why you are getting something different.

Reply:
it'll be written to config as a string, but the test is comparing the actual contents of the config so it needs to be a datetime object...

Reply:
(the test was failing due to a fun difference in how the timezone was being handled in 3.6...  see [here](https://stackoverflow.com/questions/53291250/python-3-6-datetime-strptime-returns-error-while-python-3-7-works-well))